content/en/load-testing/How-Tos/create-report-cheatsheet.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "How to use Create_report Command Options (Cheat Sheet)"
+linkTitle: "Create_report Cheat Sheet"
+
+weight: 400
+type: docs
+
+description: >
+  How to make use of different create_report command options. <!-- You will find the usage instructions and a list of available commands below. -->
+---
+## Usage
+- `create_report [<options>] <inputdir>` is used to generate a test report.
+- `create_report --help` displays all the available command options and a brief guide about their usage.
+
+### Example
+
+#### A report can be generated in the following way, without using any options.
```

Comment:
Please remove headline here, and make **Windows** and **Linux** a h4 instead (####)